english/end/end.md

```diff
@@ -0,0 +1,8 @@
+Lol this is the dead-end page that I talked to you about ;)
+
+But actually this is just a joke. I hope you enjoyed the game :)
+
+Here is a couple of options that you will wanna pick:
+
+[If you wanna eat the marshmallow, please click here lol](../eatmarshmallow/eatmarshmallow.md)
+[If you want the DreamRide, please click here](../enjoy-DreamRide/enjoy-DreamRide.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.